src/alpsParser.ts

```
@@ -1,14 +1,39 @@
+export async function parseAlpsProfile(content: string, languageId: string): Promise<DescriptorInfo[]> {
+    if (languageId === 'alps-json') {
+        return parseJsonAlpsProfile(content);
+    } else {
+        return parseXmlAlpsProfile(content);
+    }
+}
+
+async function parseJsonAlpsProfile(content: string): Promise<DescriptorInfo[]> {
+    try {
+        const jsonContent = parseJson(content);
+        if (jsonContent && jsonContent.alps && jsonContent.alps.descriptor) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)